home/views.py

```diff
@@ -46,4 +49,71 @@ def students(request):
             students_add_form.save()
             return redirect(reverse('students'))
         else:
-            return HttpResponseBadRequest('Некорректно заполнены в форме')
+            return HttpResponseBadRequest('Некорректно'
+                                          ' заполнены данные в форме')
+
+
+class StudentUpdateMain(View):
+    """
+    Redirect /student-update/ page to /students-info/
+    #:param request: output 'students-info.html'
+    #:return: prints /students-info/ page
+    """
+
+    def get(self, request): # noqa
+        return redirect(reverse('students_info'))
+
+
+class StudentUpdate(View):
+    """
+    This page updated information of students
+    #:param request:
+    #:param id: id of student in database
+    :return: updated students info page
+    """
+
+    def get_student(self, id): # noqa
+        return Student.objects.get(id=id) # noqa
+
+    def get(self, request, id): # noqa
+        student = self.get_student(id)  # noqa
+        one_student_form = StudentsAddForm(instance=student)
+        context = {'form': one_student_form, 'student_id': student.id,
+                   'student_name': student.name,
+                   'student_surname': student.surname, }
+        return render(request, 'student_update.html', context=context)
+
+    def post(self, request, id): # noqa
+        student = self.get_student(id)  # noqa
+        one_student_form = StudentsAddForm(request.POST, instance=student)
+        if one_student_form.is_valid():
+            one_student_form.save()
+            return redirect(reverse('students_info'))
+        else:
+            return HttpResponseBadRequest('Некорректно '
+                                          'заполнены данные в форме')
+
+
+class StudentsInfo(View):
+    """
+    This page print name all students in database
+    #:param request: output 'students-info.html'
+    #:return: name of all students
+     """
+
+    def get(self, request): # noqa
+        all_students = Student.objects.all()  # noqa
+        students_add_form = StudentsAddForm()
```

Comment:
rename StudentsAddForm to just StudentForm

Reply:
fixed